pkg/systemd/hw-detect.js

```diff
@@ -120,6 +120,16 @@ function findMemoryDevices(udevdb, info) {
     info.memory = memoryArray;
 }
 
+async function getBootType() {
+    const secure_boot_file = cockpit.manifests.system.config.secure_boot_file;
+    try {
+        const result = await cockpit.file(secure_boot_file, { binary: true }).read();
+        return `EFI (Secure Boot ${result[4] === 1 ? "enabled" : "disabled"})`;
+    } catch {
+        return "BIOS or Legacy";
```

Comment:
I wonder if we should check if `/proc/sys/kernel/arch` contains `x86` before reporting `BIOS`.  If we're not on x86 it might make sense to exclude this field entirely, unless we want to do some more research on this.